tests/components/application-connector/resources/charts/gateway-test/templates/applications/body-forwarding.yml

```diff
@@ -0,0 +1,38 @@
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: Application
+metadata:
+  name: body-forwarding
+  namespace: "{{ .Values.namespace }}"
+spec:
+  description: body forwarding with various methods
```

Comment:
Uppercase. I'd also keep in mind that we should somehow document the idea with the usage of `service.Description` as we use it as it is to create a request body and the method